### ISSUE TITLE
this.getDOMNode() is deprecated in react 0.14.x

### ISSUE THREAD

**Issue by beeant** [zippyui/react-radio#7]
any plan to update to react 0.14.x ?

I'm using this component and it gives me warning error

```
Warning: ReactRadioGroup.getDOMNode(...) is deprecated. Please use ReactDOM.findDOMNode(instance) instead.
```

**Comment by giulianf** [comment]
+1

**Comment by mikeluby** [comment]
+1

**Comment by Xvakin** [comment]
+1

**Comment by radubrehar** [comment]
Just released v `2.0.0` to npm which is updated for react 0.14.
Thanks for your patience.
